public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -496,14 +497,14 @@ table
 
 :marked
   ### Binding target
-  An element property between enclosing square brackets identifies the target property. The target property in the following code is the image element’s `src` property.
+  An element's target property is placed between enclosing square brackets. The target property in the following code is the image element’s `src` property.
```

Comment:
I'm not sure whether this change is correct. (The author should comment and, if possible, provide clearer wording.) Also, we avoid passive voice such as "is placed", because it hides the doer.